ui/app/styles/components/namespace-picker.scss

```diff
@@ -5,65 +5,33 @@
 
 .namespace-picker {
   position: relative;
-  color: $white;
+  color: var(--token-color-palette-neutral-300);
```

Comment:
This makes me think I should make a ticket to replace all vault colors with hds colors... just thinking out loud.

Reply:
Yes that will be a part of the HDS adoption epic work. I'm not sure if there are individual tasks for that yet or not.